### DISCUSSION TITLE
strange patterns in the interferograms from ALOS2 ScanSAR images

### DISCUSSION THREAD

**Discussion by RShamshiri** [isce-framework/isce2#566] [Q&A]
Hello everyone!

I am running alosstack on a couple of ScanSAR (VBS) images using ISCE2 (version 2.5.3). The interferograms look strange! There are parallel ramps in some sub-swaths, that there are not when processing them with SARScape! I attached one of them. I am wondering what is the reason and how can I resolve this problem!

Kind regards
Roghayeh
![ifg](https://user-images.githubusercontent.com/105787277/185580351-5d4d0b48-410b-41d3-932e-3cc2d2ec819a.png)

**Comment by fdelgadodelapuente** [comment]
Check if your data is full aperture or not. ISCE can only process ScanSAR data focused with a full aperture processor.

**Reply by RShamshiri** [reply]
Hi Francisco,

Thank you for your response. The images are full-aperture.

Kind regards,
Roghayeh
